.github/workflows/php-ci.yml

```diff
@@ -0,0 +1,25 @@
+name: PHP CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'php/**'
+      - '.github/workflows/php-ci.yml'
+  pull_request:
+    paths:
+      - 'php/**'
+      - '.github/workflows/php-ci.yml'
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: PHPUnit Tests
+        uses: php-actions/phpunit@v3
```

Comment:
@svix-onelson Is there a reason you used older-than-latest versions of both these actions?

Reply:
@svix-jplatte no.